### DISCUSSION TITLE
Persistent tabs

### DISCUSSION THREAD

**Discussion by davidstanke** [dora-team/dora.dev#253] [Announcements]
Hi all! I've just pushed a new version (thank you @jessegduke for the review!), which adds functionality for persistent tabs. So, now if you click a tab, and then copy the URL and email it to a friend, your friend will see the page the same way that you saw it.

For example: https://dora.dev/research/archives/2022/?tab_archives=questions

**Comment by jessegduke** [comment]
FYI.  You may need to clear cache if the persistent tabs are not acting as expected.
